spec/std/process_spec.cr

```diff
@@ -237,4 +237,42 @@ describe Process do
       Process.find_executable("some_very_unlikely_file_to_exist").should be_nil
     end
   end
+
+  # This doesnt work in the linux test environment due to the test environment
+  # FIXME: Discover the reason why `LibC.getgrgid(2000)` does not return a user.
+  {% if !flag?(:linux) %}
```

Comment:
Does this fail on linux generally or only on travis-ci? If it works on local runs, these tests should only be disabled for travis,